### PR TITLE
Update explainer to add the new write algorithm.

### PR DESCRIPTION
**Pull request by snianu**
Closes #????

The following tasks have been completed:

 * [x] Confirmed there are no ReSpec/BikeShed errors or warnings.
 * [X] Modified Web platform tests (link to pull request): https://github.com/web-platform-tests/wpt/pull/41722

Implementation commitment:

 * [ ] WebKit (https://bugs.webkit.org/show_bug.cgi?id=)
 * [x] Chromium (https://bugs.chromium.org/p/chromium/issues/detail?id=)
 * [ ] Gecko (https://bugzilla.mozilla.org/show_bug.cgi?id=)

### CODE REVIEW

**Review by johanneswilm — APPROVED** [review]

**Comment by johanneswilm** [comment]
@snianu @sanketj Is this still in process or should it be merged?

**Comment by snianu** [comment]
@johanneswilm Just waiting for confirmation from Chromium's security team. Once they approve it, I'll merge it, otherwise, I would have to iterate on it.

**Comment by snianu** [comment]
Chromium security team signed off these changes today so merging this PR now.

**Pull request merged**
